unitylibs/core/workflow/workflow-photoshop/workflow-photoshop.js

```diff
@@ -0,0 +1,440 @@
+import {
+  createTag,
+  getUnityLibs,
+  getHeaders,
+  loadImg,
+  createActionBtn,
+  loadSvg,
+  decorateDefaultLinkAnalytics,
+  createIntersectionObserver,
+} from '../../../scripts/utils.js';
+import { uploadAsset } from '../../steps/upload-step.js';
+import initAppConnector from '../../steps/app-connector.js';
+import createUpload from '../../steps/upload-btn.js';
```

Comment:
Can we try to import this and initialize when on demand like we did for toast

Reply:
Fixed